src/Plugins/RpcServer/Result.cs

```diff
@@ -23,7 +23,7 @@ public static class Result
         /// <typeparam name="T">The return type</typeparam>
         /// <returns>The execution result</returns>
         /// <exception cref="RpcException">The Rpc exception</exception>
-        public static T Ok_Or<T>(this Func<T> function, RpcError err, bool withData = false)
+        public static T Ok_Or<T>(Func<T> function, RpcError err, bool withData = false)
```

Comment:
why remove this `this`?

Reply:
Cause they never and will not work, cannot extend func.